### PR TITLE
Doc/edits desktop and messenger files

### PR DESCRIPTION
**Pull request by PipKat**
Getting my feet wet on topics that were already well flushed out.

### CODE REVIEW

File: pyaedt/application/MessageManager.py [hunk 1]
```diff
@@ -140,15 +135,14 @@ def messages(self):
     def get_messages(self, project_name, design_name):
         """Retrieve the Message Manager content for a specified project and design.
         
+        If the specifed project and design names are invalid, they are ignored.
```
Comment (PipKat) [inline pyaedt/application/MessageManager.py:138]:
```suggestion
        If the specified project and design names are invalid, they are ignored.
```

File: pyaedt/application/MessageManager.py [hunk 2]
```diff
@@ -19,15 +19,12 @@ class Msg():
...
-    
-    This class provides a data structure for collecting and returning messages from the 
-    AEDT Message Manager for a specified project name and design name.
+    """Collects and returns messages from the AEDT Message Manager for a specified project name and design name.
 
     Parameters
     ---------
     msg_list : list 
-        List of messages extracted from AEDT by the `AEDTMessengeManager` class.
+        List of messages extracted from AEDT by the :class:`AEDTMessengeManager` class.
```
Comment (akaszynski) [inline pyaedt/application/MessageManager.py:27]: Can't find this class.

File: pyaedt/application/MessageManager.py [hunk 3]
```diff
@@ -19,15 +19,12 @@ class Msg():
 
 
 class MessageList:
-    """MessageList class.
-    
-    This class provides a data structure for collecting and returning messages from the 
-    AEDT Message Manager for a specified project name and design name.
+    """Collects and returns messages from the AEDT Message Manager for a specified project name and design name.
```
Comment (akaszynski) [inline pyaedt/application/MessageManager.py:22]: Perfect.

**Review by akaszynski — APPROVED** [review]
One suggestion, otherwise nice work.  We're starting to converge on good docs (minus examples).

**Pull request merged**